ios/App/App/Info.plist

```diff
@@ -86,5 +86,7 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<true/>
+	<key>NSUserTrackingUsageDescription</key>
+	<string>Your data will be used to measure advertising efficiency</string>
```

Comment:
@sultanmyrza Is it possible to change the description to "We collect your data only for counting download number and will never share it with any 3rd party."

Reply:
@bafu, hope [fix(Info.plist): change NSUserTrackingUsageDescription](https://github.com/numbersprotocol/capture-lite/pull/1691/commits/815cb500700cded51b2a7afdbcfafff05fbbf7ec) commit resolves the issue